packages/reason-relay/__tests__/Test_pagination.re

```diff
@@ -79,48 +68,40 @@ module UserDisplayer = {
 module Test = {
   [@react.component]
   let make = () => {
-    let groupId = "123";
-    let query = Query.use(~variables={groupId: groupId}, ());
-
+    let userId = "123";
+    let query = Query.use(~variables={userId: userId}, ());
+    let queryRef =
+      query.node->Belt.Option.getExn.getFragmentRef_TestPagination_query();
```

Comment:
Hmm, what's this for, `getExn`?

Reply:
oh that's because the node is wrapped in an option but since the test is guaranteed to run, I just unsafely unwrapped the option. I could return `React.null` in this case if you prefer.

Reply:
Hmm, yes please, if you don't mind. Just to keep the code/test style the same across the repo.